custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/2-line_cards/card_graph.yaml

```diff
@@ -31,15 +32,15 @@ card_graph:
             - box-shadow: "none"
             - border-radius: "var(--border-radius) var(--border-radius) 0px 0px"
             - padding: "12px"
-        entity: "[[[ return variables.ulm_card_graph_entity ]]]"
```

Comment:
When multiple entities can be assigned, I think it's better to store them all in variables like (variables.entity_1; variables.entity_2; etc) to avoid confusing that one of them is not set in the variables but directly to `entity:`

Reply:
In general I agree with you for something like `chip_icon_double_state`, but I think that most of the time, the graph card will be used with only one entity. But I can revert this change if it is not wanted :)